tests/Unit/TestCreation.hpp

```diff
@@ -46,8 +46,13 @@ struct TestCreationOpt {
 };
 
 /// \ingroup TestingFrameworkGroup
-/// Construct an object from a given string. The string must not contain the tag
-/// name.
+/// Construct an object from a given string.
+///
+/// The string must not contain the
+/// name of the option (which is the name of the struct by default if no
+/// `name()` function is present). For example, to create a struct with a
+/// `SizeT` named option the following would be used:
```

Comment:
I remain confused... you wrote "the string must not contain the name of the option", but the example has an option `SizeT` and you've used the string "SizeT: 7". As I understand this, you've put the name of the option in the string. 

Is my understanding correct that in this snippet you are constructing an object of type `ClassWithoutMetavariables` where you are giving the option `SizeT` the value 7? Is it specifically "ClassWithoutMetavariables" that should be omitted from the string? If so, I would not call it an option, because as I understand our code `ClassWithoutMetavariables` is some type creatable from options, but is not itself an option.

Reply:
Okay, clearly this needed work... I pushed a fixup that hopefully answers these questions :)